docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3.6"
+services:
+  nginx:
+    image: nginx:1.17-alpine
```

Comment:
Imho the `engelsystem-nginx` could be used here?

Reply:
The prod build copies the project files into the image. In dev we may not want to have that.